lib/bindings/HTTPBinding.js

```diff
@@ -121,9 +121,9 @@ function executeCommand(apiKey, device, serializedPayload, callback) {
         options.timeout = config.getConfig().http.timeout;
```

Comment:
CHANGEX_NEXT_RELEASE entry is missing? I'd suggest to use something similar to the one you used in IOTA-UL:

```
- Add: check response obj before use it handling http commands
```

Reply:
fixed in https://github.com/telefonicaid/iotagent-json/pull/454/commits/60a22256040ef9c740d9e04aa0a69d73153fc971